install.ps1

```
@@ -0,0 +1,110 @@
+        $ActualHash = (Get-FileHash -Path $BinaryPath -Algorithm SHA256).Hash.ToLower()
+        if ($ExpectedHash -eq $ActualHash) {
+            Write-Info "Checksum verified (SHA256)"
+        } else {
+            Write-Host "error: Checksum verification failed!" -ForegroundColor Red
+            Remove-Item $BinaryPath -Force
+            exit 1
+        }
+    }
+} catch {
+    # Checksums not available; continue
+}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)